packages/@o3r/telemetry/package.json

```
@@ -92,5 +93,7 @@
     "./package.json": {
       "default": "./package.json"
     }
-  }
+  },
+  "main": "./src/public_api.js",
+  "types": "./src/public_api.d.ts"
```

I think you can target

```suggestion
  "main": "dist/src/public_api.js",
  "types": "dist/src/public_api.d.ts"
```

the `"postbuild": "patch-package-json-main"` should properly change these values in the dist/package.json